File: app/src/main/java/vn/asiantech/way/ui/profile/ProfileActivity.java
package vn.asiantech.way.ui.profile;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.design.widget.TabLayout;
import android.support.v4.view.ViewPager;
import android.support.v7.app.AppCompatActivity;

import vn.asiantech.way.R;

/**
 * Copyright © 2017 AsianTech inc.
 * Created by vinh.huynh on 12/15/17.
 */
public class ProfileActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_profile);
        TabLayout tabLayout = findViewById(R.id.htab_tabs);
        ViewPager viewPager = findViewById(R.id.htab_viewpager);
        tabLayout.setupWithViewPager(viewPager);

        MyViewPagerAdapter adapter = new MyViewPagerAdapter(getSupportFragmentManager());
        adapter.addFrag(new DummyFragment(), "FIRST");
        adapter.addFrag(new DummyFragment(), "SECOND");
        adapter.addFrag(new DummyFragment(), "THRID");
        viewPager.setAdapter(adapter);
    }
}

File: app/src/main/java/vn/asiantech/way/ui/profileanko/ProfileActivity.kt
package vn.asiantech.way.ui.profileanko

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.jetbrains.anko.setContentView

/**
 * Copyright © 2017 AsianTech inc.
 * Created by vinh.huynh on 12/15/17.
 */
class ProfileActivity : AppCompatActivity() {
    private lateinit var ui  : ProfileActivityUI
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ui = ProfileActivityUI(MyViewPagerAdapter(supportFragmentManager))
        ui.setContentView(this)
    }
}
